src/huggingface_hub/_commit_api.py

```diff
@@ -141,11 +137,17 @@ def _upload_info(self) -> UploadInfo:
         return self.upload_info
 
     @contextmanager
-    def as_file(self):
+    def as_file(self, with_tqdm: bool = False) -> Iterator[BinaryIO]:
         """
         A context manager that yields a file-like object allowing to read the underlying
         data behind `path_or_fileobj`.
 
+        Args:
+            with_tqdm (bool, *optional*):
```

Comment:
I think the convention is to have `defaults to False`, no?

Reply:
:tired_face:

Reply:
Still don't know about that. I'm seing a lot of `private (`bool`, *optional*, defaults to `False`):` so I'll put that.
I remember a conversation about that on Slack, wasn't so clear what the output is though :confused: